converter/templates/index.html

```diff
@@ -1,74 +1,327 @@
-        </li>   
-        
-        <li id="text-box"style="background:rgb(0, 248, 165);height: 300px;width: 240px;border-width: 10px;border-color: white;">
+        <div id = "text" style="text-align: center">
+            <form method="POST" id="text_input">
+                <textarea name="content" id = "textcontent" form="text_input" rows="8" cols="80"></textarea>
+                <button type ="submit" class="btn btn-primary btn-block" id = "text_submit">Submit</button>
+            </form>
+        </div>
         
-            <h1 style="padding: 300px 0px 0px 0px;" id="text-button" class="submittext">Submit Text</h1>
-        </li>  
+        <br>
 
-    </ul>
-    <br style="padding: 30px 0px 0px 0px;">
-
-    <div id = "text" style="text-align: center">
-        <form action="/convert_result/submission" method="POST" id="input">
-            <textarea name="content" form="input" rows="8" cols="50"></textarea>
-            <input type="submit" value="Submit">
-        </form>
+        <div id = "file">
+            <form method="POST" enctype="multipart/form-data" class="box">
+                <div class="box__input">
+                    <svg class="box__icon" xmlns="http://www.w3.org/2000/svg" width="50" height="43" viewBox="0 0 50 43"><path d="M48.4 26.5c-.9 0-1.7.7-1.7 1.7v11.6h-43.3v-11.6c0-.9-.7-1.7-1.7-1.7s-1.7.7-1.7 1.7v13.2c0 .9.7 1.7 1.7 1.7h46.7c.9 0 1.7-.7 1.7-1.7v-13.2c0-1-.7-1.7-1.7-1.7zm-24.5 6.1c.3.3.8.5 1.2.5.4 0 .9-.2 1.2-.5l10-11.6c.7-.7.7-1.7 0-2.4s-1.7-.7-2.4 0l-7.1 8.3v-25.3c0-.9-.7-1.7-1.7-1.7s-1.7.7-1.7 1.7v25.3l-7.1-8.3c-.7-.7-1.7-.7-2.4 0s-.7 1.7 0 2.4l10 11.6z"></path></svg>
+                    <input type="file" name="files" id = "filel" class="box__file">
+                    <label for="filel"><strong>Choose a file</strong><span class="box__dragndrop"> or drag it here</span>.</label>
+                    <button type="submit" class="btn btn-success btn-block box__button">Upload</button>
+                </div>
+            </form>
+        </div>
     </div>
+    <div id = "result_page">    
+        <div style="text-align : center;">
+            <textarea name="content" id="result_content" rows="15" cols="80"></textarea>
+            <br>
+        </div>
     
-    <br>
-
-    <div id = "file">
-        <form action="/convert_result/upload" method="POST" enctype="multipart/form-data" style="text-align: center;">
-            <input type="file" name="file">
-            <input type="submit" value="Upload">
-        </form>
+        <div>
+            <form action='/'>
+                <input class="btn btn-primary btn-block" type="submit" value="Return">
+            </form>
+            <br>
+        </div>
+    
+        <div>
+            <form action='/return-files'>
+                <input class="btn btn-success btn-block" type="submit" value="Download" id = "download">
+            </form>
+        </div>
     </div>
 
 </div>
 
     <script>
```

Comment:
Once the functionality is confirmed, all scripts should be refactored into .js files and stored in separate directories.